internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -169,3 +216,34 @@ func parseExpose(line string) []portConfig {
 	}
 	return ports
 }
+
+func parseHealthCheck(line string) (healthCheck, error) {
+	var hc healthCheck
+
+	if line[12:] == "NONE" {
+		return hc, nil
+	}
+
+	var retries int
+	var interval, timeout, startPeriod time.Duration
+	fs := flag.NewFlagSet("flags", flag.ContinueOnError)
+
+	// Default value DurationVar gives is in nanoseconds
```

Comment:
Such a creative implementation! I like it. Maybe making `interval`... to be consts as well so that folks will know they are actually option names for health check instruction.

Reply:
@efekarakus He showed me a great example in my design doc!